.github/workflows/publish-docker.yml

```diff
@@ -58,12 +63,12 @@ jobs:
           docker image ls
       - name: Log in to the Container registry
         uses: docker/login-action@v1.10.0
-        if: github.ref == 'refs/heads/main'
+        # if: github.ref == 'refs/heads/main'
         with:
           registry: ${{ env.HUB }}
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
       - name: Push docker image
-        if: github.ref == 'refs/heads/main'
+        # if: github.ref == 'refs/heads/main'
```

Comment:
TODO, are these temp changing?

Reply:
It's only for testing and this PR is not ready for review